.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,29 +1,46 @@
-## Purpose
-<!--- Describe the problems, issues, or needs driving this feature/fix and include links to related issues -->
-The purpose of this PR is to fix #<issue-number>
+---
+labels: pending-review
+assignees:
+  - jayasanka-sack
+  - akshika47
+  - anjula-sack
+  - Piumal1999
```

Comment:
Why do we need this?

Reply:
It will automatically assign PR to the given users. Unlike other repos, handbook PRs are important as the proposed changes directly impact the organisational workflows, therefore better to assign us so that it will notify us when someone proposes a change. More info: [How to change or define a process](https://handbook.sefglobal.org/organisation/handbook/usage#how-to-change-or-define-a-process)

Reply:
Nice